src/ApparentHorizons/StrahlkorperGr.cpp

```diff
@@ -202,20 +202,22 @@ Scalar<DataVector> spin_function(
       make_with_value<Scalar<DataVector>>(area_element, 0.0);
   for (size_t i = 0; i < 3; ++i) {
     for (size_t j = 0; j < 3; ++j) {
+      const DataVector extrinsic_curvature_normal_sin_theta =
+          extrinsic_curvature.get(i, j) * unit_normal_vector.get(j) *
+          get(sin_theta);
+
       // Note: I must multiply by sin_theta here because
       // I take the phi derivative of this term by using
       // the spherepack gradient, which includes a
       // sin_theta in the denominator of the phi derivative.
       get(extrinsic_curvature_theta_normal_sin_theta) +=
-          extrinsic_curvature.get(i, j) * tangents.get(i, 0) *
-          unit_normal_vector.get(j) * get(sin_theta);
+          extrinsic_curvature_normal_sin_theta * tangents.get(i, 0);
 
       // Note: I must multiply by sin_theta here because tangents.get(i,1)
       // actually contains \partial_\phi / sin(theta), but I want just
       //\partial_\phi.
       get(extrinsic_curvature_phi_normal) +=
-          extrinsic_curvature.get(i, j) * tangents.get(i, 1) *
-          get(sin_theta) * unit_normal_vector.get(j);
+          extrinsic_curvature_normal_sin_theta * tangents.get(i, 1);
     }
   }
 
```

Comment:
now after both i and j loops, multiply the the two Scalars by sin_theta (moving your comments about sin theta)